docs/source/configuration/persisted-queries.mdx

```diff
@@ -66,6 +66,21 @@ persisted_queries:
 
 If used with the [`safelist`](#safelist) option, the router logs unregistered and rejected operations. With [`safelist.required_id`](#require_id) off, the only rejected operations are unregistered ones. If [`safelist.required_id`](#require_id) is turned on, operations can be rejected even when registered because they use operation IDs rather than operation strings.
 
+#### `experimental_local_manifests`
+
+<ExperimentalFeature />
+
+Adding `experimental_local_manifests` to your `persisted-queries` configuration lets you use local persisted query manifests instead of the hosted Uplink version. This is helpful when you're using an offline Enterprise license and can't use Uplink. With the  `experimental_local_manifests`, the router doesn't reload the manifest from the file system, so you need to restart the router to apply changes.
```

Comment:
`s/-/_/`.

```suggestion
Adding `experimental_local_manifests` to your `persisted_queries` configuration lets you use local persisted query manifests instead of the hosted Uplink version. This is helpful when you're using an offline Enterprise license and can't use Uplink. With the `experimental_local_manifests` option, the router doesn't reload the manifest from the file system, so you need to restart the router to apply changes.
```